### PR TITLE
updated license script and readme

### PR DESCRIPTION
**Pull request by schiessle**
This pull request introduced following changes:
# license.php
- we keep the copyright statement and only update the authors list. Background: Not every line change is copyrighted so we can't give everyone copyright, just because git blame says that a line was changed by a specific user. This also allows us to add copyright statements to the files next to ownCloud Inc without losing it the next time we run the script
- The script checks the timestamp of each line change. If all lines of a file where changed after 6.6.2016 (the day we forked ownCloud) it will produce a output which allows us to have a look at the file and decide if we can re-license it to AGPLv3 or later.
# readme.md
- add a statement that every contributions from now on are considered to be licensed under the AGPLv3 or later.
-  as explained above, it is not a good idea to change the copyright statement automatically. Therefore we encourage contributors to add a `@copyright ...` line with their name to any file they changed in a substantial way. The script will make sure that everyone is listed as a author, but everyone can and have to decide by its own if his changes are worth a copyright notice.

### CODE REVIEW

**Comment by schiessle** [comment]
@jospoortvliet @karlitschek please also have a look at this PR, especially the readme.md changes

File: README.md [hunk 1]
```diff
@@ -30,7 +30,20 @@ if you want to join the Github organization just let us know and we’ll add you
...
 
 ## Contribution Guidelines
-https://nextcloud.com/contribute/
+
+All contributions to this repository from June, 16 2016 on are considered to be 
+licensed under the AGPLv3 or any later version.
+
+Nextcloud doesn't require a CLA (Contributor License Agreement).
+The copyright belongs to all the individual contributors. Therefore we recommend 
+that every  contributor adds following line to the header of a file, if he
```
Comment (nickvergessen) [inline README.md]: two spaces after `every`

Comment (schiessle) [inline README.md]: fixed

**Comment by karlitschek** [comment]
looks good 👍

**Comment by rullzer** [comment]
You got to be careful with that date check.
We have merged oC master back here. And might cherry pick commits in the future.

**Comment by schiessle** [comment]
> You got to be careful with that date check.
> We have merged oC master back here. And might cherry pick commits in the future.

Yes, that's why I don't change anything by default. It just output a list of files where you should have a look. If you don't change it you get the same message again one year later the next time you run the script.

**Comment by rullzer** [comment]
Fair enough.
Just wanted to make extra sure ;)

:+1:

**Pull request merged**